src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/VariableNamesShouldNotMatchFieldNames.cs

```diff
@@ -13,6 +13,7 @@ namespace Microsoft.CodeQuality.Analyzers.Maintainability
     public abstract class VariableNamesShouldNotMatchFieldNamesAnalyzer : DiagnosticAnalyzer
     {
         internal const string RuleId = "CA1500";
+        private const string HelpLinkUri = "https://msdn.microsoft.com/en-us/library/ms182216.aspx";
```

Comment:
❗️ https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1500-variable-names-should-not-match-field-names